### PR TITLE
Move use cases to bikeshed

### PR DESCRIPTION
**Pull request by g-ortuno**
Old: https://rawgit.com/WebBluetoothCG/web-bluetooth/master/use-cases.html#security_privacy

New: https://api.csswg.org/bikeshed/?url=https://github.com/g-ortuno/web-bluetooth/raw/move-use-cases-to-bikeshed/use-cases.bs

### CODE REVIEW

**Comment by g-ortuno** [comment]
Wait! It's not ready yet. I need to fix the Table of Contents.

**Comment by g-ortuno** [comment]
@scheib PTAL. I split the change into several commits for easier reviewing.

File: use-cases.bs [hunk 1]
```diff
@@ -1,69 +1,31 @@
...
-          wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/72794/status",
-
-          otherLinks: [{
-            key: "Participate",
-            data: [
-              { value: "Join the W3C Community Group",
-                href: "http://www.w3.org/community/web-bluetooth/",
-              },
-              { value: "File bugs and feature requests",
-                href: "https://github.com/WebBluetoothCG/web-bluetooth/issues",
```
Comment (scheib) [inline use-cases.bs]: Issues are 'inline' in the bikeshed version now, would be ideal to have the github location still listed. 
https://tabatkins.github.io/bikeshed/#metadata-issue-tracking mentions this, but I don't see it used by our main spec. https://webbluetoothcg.github.io/web-bluetooth -- it seems that it is implicit and [magical](https://github.com/tabatkins/bikeshed/issues/860).

**Comment by scheib** [comment]
Note, github makes it hard to see the comment I left on the old version of use-csaes.bs -- you'll see it in "Hide outdated".

File: use-cases.bs (hunk 1 above)
Comment (g-ortuno) [inline use-cases.bs]: hmm I think that might be a bug. The generated file on my machine includes the Github link. I commented on the issue you opened. I'll land this and fix the file depending on the outcome of the issue.

**Pull request merged**